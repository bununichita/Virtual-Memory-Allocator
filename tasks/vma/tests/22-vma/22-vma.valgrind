==20489== Memcheck, a memory error detector
==20489== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==20489== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==20489== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==20489== Parent PID: 20488
==20489== 
==20489== 
==20489== HEAP SUMMARY:
==20489==     in use at exit: 0 bytes in 0 blocks
==20489==   total heap usage: 12 allocs, 12 frees, 8,392 bytes allocated
==20489== 
==20489== All heap blocks were freed -- no leaks are possible
==20489== 
==20489== For lists of detected and suppressed errors, rerun with: -s
==20489== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
