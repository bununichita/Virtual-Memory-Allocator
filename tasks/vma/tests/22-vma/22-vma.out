GIGEL
1
5
